tests/sles4sap/netweaver_filesystems.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's SLES4SAP openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Configure NetWeaver filesystems
+# Maintainer: Loic Devulder <ldevulder@suse.de>
+
+use base "sles4sap";
+use testapi;
+use utils 'systemctl';
+use hacluster;
+use strict;
+
+sub run {
+    my ($self)      = @_;
+    my $type        = get_required_var('INSTANCE_TYPE');
+    my $instance_id = get_required_var('INSTANCE_ID');
+    my $sid     = get_var('INSTANCE_SID', 'HA1');
+    my $arch    = get_required_var('ARCH');
+    my $lun     = get_lun;
+    my $sap_dir = "/usr/sap/$sid";
+    my ($proto, $path) = $self->fix_path(get_required_var('NW'));
+
+    # LUN information is needed after for the HA configuration
+    set_var('INSTANCE_LUN', "$lun");
+
+    select_console 'root-console';
+
+    # Create/format/mount local filesystems
+    assert_script_run "mkfs -t xfs -f $lun";
+    foreach my $mountpoint ("${type}${instance_id}", 'SYS') {
+        # Mountpoints must be created on all nodes
+        assert_script_run "mkdir -p $sap_dir/$mountpoint";
+    }
```

Comment:
I think you can drop this `foreach` block and just leave the `mkdir` call for `$sap_dir/${type}${instance_id}` since you will call `mkdir` on `SYS` again on L48 below.

Reply:
That was more complicated before, but yes not very useful anymore.